docs/site/index/cldr-spec/picking-the-right-language-code.md

```diff
@@ -1,92 +1,92 @@
+
+## Choosing the Base Language Code
+
+1. Go to [iso639\-3](https://iso639-3.sil.org/code_tables/639/data) to find the language. In the "Language Name(s) containing" field enter the language name (**Ganda** in this example) and select "Apply".
+2. There may be multiple entries for the item you want, so you'll need to examine them carefully.
+3. You'll find an entry like:
+
+&emsp;lug&emsp; lug&emsp; **lg**&emsp; Ganda&emsp; Individual&emsp; Living
+
+While you may think that you are done, you have to verify that the three\-letter code is correct.
+
+1. Click on the item in the first column ([lug](https://iso639-3.sil.org/code/lug) in this case) and you'll find links to Ethnologue, Glottolog and Wikipedia.
+2. Click on the [Ethnologue](http://www.ethnologue.com/language/lug) link and you get to the corresponding Ethnologue entry.
+3. Verify that is indeed the language:
+	1. Look at the information on the Ethnologue page
+	2. Check Glottolog, Wikipedia and other web sources
+4. ***AND IMPORTANTLY: Review "[Caution](/index/cldr-spec/picking-the-right-language-code#caution)" section below***
+
+Once you have the right three\-letter code, you are still not done. Unicode and BCP 47 use the two\-letter ISO code if it exists. Unicode also uses the "macro language" where suitable. *So*
+
+1. Use the two\-letter code if there is one. In the example above, use the highlighted "lg" from the first table.
+2. Verify that the code is in the [IANA subtag registry](http://www.iana.org/assignments/language-subtag-registry)
+3. If the code occurs in the [supplementalMetadata.xml](http://github.com/unicode-org/cldr/blob/main/common/supplemental/supplementalMetadata.xml) file in the type attribute of a languageAlias element, then use the replacement instead.
+	- For example, because "swh" occurs in \<languageAlias type\="swh" replacement\="sw"/\>, "sw" must be used instead of "swh".
+
+## Choosing Script/Territory Subtags
+
+If you need a particular variant of a language, then you'll add additional subtags, typically script or territory. Consult [Sample Subtags](http://unicode.org/cldr/utility/sample_subtags.html) for the most common choices. ***Again, review "[Caution](/index/cldr-spec/picking-the-right-language-code#caution)" section below.***
+
+## Verifying Your Choice
+
+1. Verify your choice by using the [online language identifier](http://unicode.org/cldr/utility/languageid.jsp) demo.
+2. You need to fix the identifier and try again in *any* if the demo shows any of the following:
+	1. the language identifer is illegal, or
+	2. one of the subtags is invalid, or
+	3. there are any replacement values.
+
+## Documenting Your Choice
+
+If you are requesting a new locale / language in CLDR, please include the links to the particular pages above so that we can process your request more quickly, as we have to double check before any addition. The links will be of the form:

```

Comment:
```suggestion
If you are requesting a new locale / language in CLDR, please include the links to the particular pages above so that we can process your request more quickly, as we have to double-check the information before making the addition. The links will be of the form:

```